app/views/registrations/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Create Account</h2>
+
+<%= devise_error_messages! %>
+
+<%# View for Devise method inherited by Registrations controller. %>
+<%# Uses `resource` for consistency with Devise::Registrations#new %>
+<%= form_for(resource,
+             as: resource_name,
+             url: registrations_path(resource),
+             html: { class: 'pss-admin-form' }) do |f| %>
+
+  <%= f.label :email %>
+  <%= f.email_field :email, autofocus: true %>
+
+  <%= f.label :password %>
+  <%= f.password_field :password, autocomplete: "off" %>
+  <% if @validatable %>
+  <div><em>(<%= @minimum_password_length %> characters minimum)</em></div>
+  <% end %>
+
+  <%= f.label :password_confirmation %>
+  <%= f.password_field :password_confirmation, autocomplete: "off" %>
```

Comment:
Can we remove the ability to add a password to this form?  

I found that if I create a new account with a password, it gets really confusing for the new user:  The confirmation email is sent to the new email address.  Then, from the email, when I click "Confirm my account", I am taken, already signed in, to `primary-source-sets/sets`.  I never learn my password, so if I get logged out before I change it, I can't get back in again, and my confirmation token expires.

Plus, I don't think people will want to admins create new accounts and set passwords for those accounts, correct?